packages/webamp-modern-2/src/maki/interpreter.ts

```diff
@@ -312,7 +316,32 @@ class Interpreter {
               obj.value != null,
             `Guru Meditation: Tried to call method ${klass.name}.${methodName} on null object`
           );
-          let value = obj.value[methodName](...methodArgs);
+          // let value = obj.value[methodName](...methodArgs);
+          let value = null;
+          try {
+            if (obj.value[methodName]) {
+              value = obj.value[methodName](...methodArgs);
+            } else {
+              value = obj.value.constructor[methodName](...methodArgs);
+            }
+          } catch (err) {
+            value = null;
+
+            try {
+              const fun = (
+                obj.value[methodName]
+                  ? obj.value[methodName]
+                  : obj.value.constructor[methodName]
+              ).bind(obj.value);
```

Comment:
This is confusing to me. So, here we are trying again, but this time with the function bound to `obj.value`? Are there currently cases where that's actually needed?

I'm especially curious about the case there `obj.value[methodName]` is true. Is there ever a case where `obj.value[methodName].bind(obj.value)(...methodArgs)` returns a different value than `obj.value[methodName](...methodArgs)`?

Reply:
so please, solve the problem of calling "Config" as const by maki (above). 
Currently Config is a class, not an instance of class, but maki threat it as a global variable name.

Reply:
do you think that revert the hack is good? 
I think it is good to remove this my brutal solution :p

Reply:
Yeah. I am happy, I removed them all. It is finally less cluttered than I ever imagined.
Still it leaves a warning, but it is now unrelated to Config/ConfigAttributes etc.
```
error call: Layer.setxmlparam(...["dblclickaction","switch;shade"]) err: obj.value[methodName] is not a function obj: {"type":"OBJECT","value":{"global":1,"type":"INT","value":0},"global":1,"guid":"5ab9fa1545579a7d5765c8aba97cc6a6"}
```